src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -159,13 +207,24 @@ protected internal override void SetPinMode(int pinNumber, PinMode mode)
             int requestResult = -1;
             if (_pinNumberToSafeLineHandle.TryGetValue(pinNumber, out SafeLineHandle pinHandle))
             {
-                if (mode == PinMode.Input)
-                {
-                    requestResult = Interop.libgpiod.gpiod_line_request_input(pinHandle, s_consumerName);
-                }
-                else
+                int flags;
+
+                switch (mode)
                 {
-                    requestResult = Interop.libgpiod.gpiod_line_request_output(pinHandle, s_consumerName);
+                    case PinMode.Input:
+                        requestResult = Interop.libgpiod.gpiod_line_request_input(pinHandle, s_consumerName);
+                        break;
+                    case PinMode.InputPullDown:
+                        flags = (int)RequestFlag.GPIOD_LINE_REQUEST_FLAG_BIAS_PULL_DOWN;
```

Comment:
Doesn't seem to need this `flag` variable, for me better to just use the enum below inline

Reply:
Thanks @buyaa-n fixed